CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Popover: Added `focusAnchorOnClose` prop to reset the focus back to the anchor element when popover closes and focus is set within. Default to `true`;
+- Dropdown: Added `focusAnchorOnClose` prop to reset the focus back to the anchor element when popover closes and focus is set within. Default to `true`;
+- List: Added `tabIndex` prop to control the list tabIndex. Default to `-1` to avoid breaking changes.
```

Comment:
Why is it avoiding breaking change to default to `-1` instead of `undefined` ?

Reply:
Because I believe some components, like the Dropdown I think, sets the focus on the List when it appears.

If the tabIndex is undefined, this simply wont work. You need to add `tabIndex={-1}` in order to programmatically set a focus on a element.